doc/api/fs.md

```diff
@@ -1767,7 +1767,7 @@ changes:
   * `encoding` {string|null} default = `null`
   * `flag` {string} default = `'r'`
 
-Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
+Synchronous version of [`fs.readFile()`][]. Returns the contents of the `file`.
```

Comment:
Done for this and another place. It seems these typos are from the same source as the link issues.